CHANGELOG

```diff
@@ -1,3 +1,17 @@
+pcmanfm-qt-0.14.0 / 2019-
```

Comment:
Should this be 0.14.1?

Also no date?  (Same for lxqt-config-0.14.1 it seems.)

Reply:
it should - and i wanted to do some tests first before i set the date

Reply:
thanks a lot, damn typos :D

Reply:
Alrighty!  Figured the date was just not set yet, but wasn't sure.  :]